src/libraries/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -247,6 +247,10 @@ namespace System.Drawing
         public static Color PowderBlue => new Color(KnownColor.PowderBlue);
 
         public static Color Purple => new Color(KnownColor.Purple);
+        
+        public static Color RebeccaPurple => new Color(KnownColor.RebeccaPurple);
+
+        public static Color RebeccaPurple => new Color(KnownColor.RebeccaPurple);
```

Comment:
This is duplicated, missed to remove the previous one.

Reply:
fixed thanks